docs/book/service-manager.md

```diff
@@ -110,6 +110,23 @@ Because the main filter is `Priority`, it can be set directly too:
 ];
 ```
 
+## PsrLoggerAbstractServiceFactory
+
+Same as above, you can use PsrLoggerAbstractServiceFactory to create a PSR-3 conform logger.
+Just use following configuration instead.
```

Comment:
"just", "simple", "easy" etc. should be avoided in a (technical) documentation. It's hard to say if it's _really_ easy or simple for everyone.